docs/governance/community-groups.md

```diff
@@ -0,0 +1,27 @@
+# Common Cloud Controls (CCC) Community Structure
+
+The CCC community is organized into a structure that enables contributors from diverse backgrounds to collaborate on developing the Common Cloud Controls standards. This structure ensures efficient governance, clear guidance for contributors, and effective management of the project's technical and non-technical aspects.
+
+All community group participants must strictly adhere to the the [FINOS Code of Conduct].
+
+## Steering Committee
+
+The **Steering Committee (SC)** is the ultimate governance body of the CCC project, responsible for strategic oversight, financial planning, defining the project's values, high-level decision-making, and coordination with the FINOS TOC or Board. It also handles the overall project structure and delegation of responsibilities to various groups.
```

Comment:
Maybe remove "ultimate". Doesn't feel very open sourcey and the Board is the ultimate governance for FINOS.